### ISSUE TITLE
Why RGB values on transparent pixels is (71,112,76)

### ISSUE THREAD

**Issue by shanegxxiao** [kornelski/pngquant#362]
After compare several pairs images processed by pngquant, it seems pixel with rgba = (0, 0, 0, 0) is modified to (71, 112, 76, 0), How is it happen? It make premultiply alpha works failed.

**Comment by kornelski** [comment]
No, it's not a failure. When alpha=0 these values have no meaning. They're visible only in software that incorrectly drops the alpha channel, and in that case they're useful for diagnosing the error.
